### PR TITLE
Disable apply button when there's a approved application

### LINKED ISSUE #143: Apply button should be disabled when there's a approved application
**Is your feature request related to a problem? Please describe.**
Apply button should be disabled when there's a approved application.

**Describe the solution you'd like**
The apply button should be disabled when there's a approved application with a tooltip mentioning you can apply only for one mentor at a time.

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by Madhawa97**
## Purpose

The purpose of this PR is to fix #143
## Goals

Disable apply button when there's a approved application

## Approach

- Disable apply button when there's a approved application similarly to pending state.
- Invalidate profile upon successful submission

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/pages/MenteeRegistration/index.tsx [hunk 1]
```diff
@@ -126,6 +126,9 @@ const MenteeRegistration: React.FC = () => {
         { withCredentials: true }
       );
     },
+    onSuccess: async () => {
+      await refetch();
```
Comment (anjula-sack) [inline src/pages/MenteeRegistration/index.tsx:130]: Let's invalidate the query instead of  refetch

**Review by anjula-sack — CHANGES_REQUESTED** [review]

File: src/pages/MenteeRegistration/index.tsx (hunk 1 above)
Comment (Madhawa97) [reply src/pages/MenteeRegistration/index.tsx:130]: That's what `refetch` method does, isn't? It was written in `useProfile`.

Comment (anjula-sack) [reply src/pages/MenteeRegistration/index.tsx:130]: oh yes, my bad

**Review by anjula-sack — APPROVED** [review]
Thank you! @Madhawa97

File: src/pages/MenteeRegistration/index.tsx (hunk 1 above)
Comment (Madhawa97) [reply src/pages/MenteeRegistration/index.tsx:130]: No worries 🙌🏼

**Pull request merged**